extras/module-node/Makefile

```diff
@@ -14,3 +14,4 @@ test:
 	./test 'var ape = require("ape"); ape.module.filename === "ape.js" ? "module.filename OK" : "module.filename FAILED";'
 	./test 'var ape = require("ape"); ape.module.loaded === true && ape.wasLoaded === false ? "module.loaded OK" : "module.loaded FAILED";'
 	./test 'var ape = require("ape"); ape.__filename === "ape.js" ? "__filename OK" : "__filename FAILED";'
+	./test 'var badger = require("badger"); badger.foo === 123 && badger.bar === 234;'
```

Comment:
Oops, that's actually not enough because the 'make test' is now depending on exit code, which gets set to an error only if an error is thrown. That is, if one wants to run `make test` and assume that exit code 0 means everything is working. (None of the other extras do that, but that'd be a simple way to have at least some coverage.)